dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -141,9 +140,9 @@ object WavesDexCli extends ScoptImplicits {
              |  Account seed          : ${args.accountSeed.getOrElse("")}
                    """.stripMargin
         )
-        (config, matcherConfig) <- cli.wrapByLogs("  Loading Matcher settings... ")(loadAllConfigs(args.configPath))
-        superConnector <- SuperConnector.create(matcherConfig, apiUrl, args.nodeRestApi, args.authServiceRestApi, apiKey)
-        checkResult <- new Checker(superConnector).checkState(args.version, args.accountSeed, config, matcherConfig)
+        (config, matcherSettings) <- cli.wrapByLogs("  Loading Matcher settings... ")(loadAllConfigs(args.configPath))
```

Comment:
It looks like we ca move `matcherSettings` to the `checkServer` parameters.

Reply:
resolved